releases/v1.2.3.toml

```diff
@@ -0,0 +1,31 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.2.2"
+
+pre_release = false
+
+preface = """\
+### Notable Updates
+* fix for Tar xattrs [#2953](https://github.com/containerd/containerd/pull/2953)
```

Comment:
While true, it doesn't show the magnitude of the fix.

Maybe something like:
* restores compatibility with older container images [#2953](https://github.com/containerd/containerd/pull/2953)
?